### DISCUSSION TITLE
Coal not usable as fuel in most appliances

### DISCUSSION THREAD

**Discussion by RodriguesCIA** [cataclysmbn/Cataclysm-BN#2062] [General]
Coal can't be used in the built smoker and rock forge, but works with charcoal smoker and charcoal forge (the carriable item). Not sure about the charcoal cooker, but I think it doesn't work with it either.

This might usually be a small issue for most playthroughs, but as I made conscious effort of using black gunpowder (Historical Reenactor) in early game, and raided a few mines early on, I gathered a staggering amount of coal that only when I needed the most I found out I couldn't use at all. While you can make large amounts of charcoal, the stack you get from a mine (around 10k?) probably beats what you could make by actively paying attention to a kiln for multiple days.

On recipes, seems to work well enough as a charcoal alternative, just not as a fuel.

**Comment by Proxiehunter** [comment]
I'm not sure coal should work with _either_ smoker, see [here](https://www.quora.com/What-happens-if-I-grill-with-coal-instead-of-charcoal) for someone confirming my suspicions on that front. It should definitely work in the rock forge though. Although it might produce nasty gasses that you would want your survivor wearing some sort of breathing mask for. Coal and charcoal are two different things and can not be used the same way.

**Comment by chaosvolt** [comment]
Basically, this is an issue with reloadable furniture. Instead of being able to look at the ammotype it's supposed to be loaded with, it's only able to accept that ammotype's default ammo, which in the case of the coal/charcoal ammotype is charcoal.

I have no idea how complicated it'd be to fix C++wise, and the only JSON level fix would be something in the vein of changing charcoal's ammotype to use a generic fuel item as its default, which is then made craftable from charcoal or coal.
